data/recommendation/notify.js

```diff
@@ -60,6 +64,10 @@ class Notify {
   showSyncSignup() {
     document.querySelector('#sync').classList.remove('hidden');
   }
+
+  disableForever() {
```

Comment:
You don't use this, you should either get rid of it or pass it to the `addEventListener` call instead of the anonymous function.